apps/admin-x-settings/src/components/settings/site/designAndBranding/ThemeSettings.tsx

```diff
@@ -70,11 +68,9 @@ const ThemeSettings: React.FC<ThemeSettingsProps> = ({sections, updateSetting})
 
                             // hides typography related theme settings from official themes
                             // should be removed once we remove the settings from the themes in 6.0
-                            if (hasCustomFonts) {
-                                const hidingSettings = themeSettingsMap[activeThemeName];
-                                if (hidingSettings && hidingSettings.includes(setting.key) && activeThemeAuthor === 'Ghost Foundation' && supportsCustomFonts) {
-                                    spaceClass += ' hidden';
-                                }
+                            const hidingSettings = themeSettingsMap[activeThemeName];
+                            if (hidingSettings && hidingSettings.includes(setting.key) && activeThemeAuthor === 'Ghost Foundation' && supportsCustomFonts) {
```

Comment:
So we should be hiding the block if the theme doesn't support it, based on [this](https://github.com/TryGhost/Ghost/blob/main/apps/admin-x-settings/src/hooks/useCustomFonts.tsx#L7) line. Then it makes sense to not have an error message.

Reply:
Actually, this block is confusing to me. Why are we only hiding the settings for official themes?

Reply:
We have font-related custom font settings on almost all our official themes. We want to hide them when official themes support the new custom fonts feature, because it'd be confusing for users to have two font-related settings — one in the brand settings, and the other one in the theme settings.